docs/rfcs/9999-vpn-firefox-integration.md

```diff
@@ -0,0 +1,164 @@
+- Status: Draft
+- Date: 2023-05-29
+- Authors:
+    - [@strseb](https://github.com/strseb)
+    - [@lesleyjanenorton](https://github.com/lesleyjanenorton)
+    - [@oskirby](https://github.com/oskirby)
+- RFC PR: -
+- Implementation GitHub issue: -
+
+-----
+
+# Problem Statement
+
+
+![](https://github.com/mozilla-mobile/mozilla-vpn-client/assets/9611612/aa8331cf-f0cb-489b-a7e2-293bce422551) 
+
+VPNs currently allow you to select the country through which you want your traffic to emerge to the Internet and users use this to express a variety of geolocation preferences. These controls are not flexible enough - users can only select a single location through which all traffic should go through. 
+
+Therefore we want to propose a new Web-Extension that extends on top of the VPN+MAC API, giving users the options to:
+ - Enable The VPN only for Firefox 
+ - Disable The VPN only for Firefox 
+ - Exclude a Page from VPN protection
+ - Specify an Exit Location a Page.
+
+We already have integrated into Firefox with [Multi Account Containers](https://github.com/mozilla/multi-account-containers) which allows users to specify an exit location for a container. 
+The following parts expect knowledge on how the MAC Extension works, for a refresher scroll to appendix :) 
+

```

Comment:
The Appendix describes the role of the VPN Client. What happens on a mobile platform where the VPN is active but the client has been closed?

Reply:
the MAC integration is not supported on mobile :) , added that to the appendix.